applications: added new endpoint(smart-plug) to matter-bridge (#16498)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

  * The :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILES_PATH` Kconfig option, which specifies ZAP files location for the application.
    By default, the option points to the :file:`src/default_zap` directory and can be changed to any path relative to application's location that contains the ZAP file and :file:`zap-generated` directory.
  * Support for the :ref:`zephyr:nrf54h20dk_nrf54h20`.

>   * Optional ability to work as smart-plug device.

Reviewer: ```suggestion
  * Optional functionality to work as a smart plug device.
```

>   * The smart plug device functionality (optional).

Reviewer 2: ```suggestion
  * Optional smart plug device functionality.
```

>   * Optional functionality to work as a smart plug device.

Reviewer: ```suggestion
  * The smart plug device functionality (optional).
```
I know I've checked this before but just thought that maybe this sounds more neat?
